src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, num) {
+      callback.call(this, num);
+
+      return this;
```

Comment:
The `operate` method correctly uses the callback pattern and allows chaining by returning `this`. Good use of method shorthand syntax.